api_tests/lib/btsieve_runner.ts

```diff
@@ -0,0 +1,56 @@
+import { ChildProcess, spawn } from "child_process";
+import { MetaBtsieveConfig } from "./btsieve";
+import * as fs from "fs";
+
+export class BtsieveRunner {
+    running_btsieves: { [key: string]: ChildProcess };
```

Comment:
🙃 Should we go camelCase?

Reply:
I would rather not for now as we use a mix in between camel and snake_case... maybe a good point to bring up for a new linter?